includes/admin/views/omise-page-settings.php

```diff
@@ -16,7 +34,26 @@
 		);
 		?>
 	</p>
+
 	<form method="POST">
+		<!-- Section: account information -->
+		<?php if ( $settings['account_email'] ) : ?>
```

Comment:
Why only check for `account_email` if you are also displaying `account_country`?

Reply:
We may check `account_country` as well but I just think it is unnecessary.
This row will be displayed only when Omise account has been set, so I was looking for a variable that can be used as condition for this case. It will either have its value, or just empty string.
Default case:
```php
echo $settings['account_email']; // ""
echo $settings['account_country']; // ""
```

After connect with Omise account
```php
echo $settings['account_email']; // "...@omise.co"
echo $settings['account_country']; // "TH"
```

For the rare case (that I'm not sure how could it happen), it might be as this
```php
echo $settings['account_email']; // "...@omise.co"
echo $settings['account_country']; // ""
```
However, it's still safe to print `$settings['account_country']` out as the default of this variable is empty string.

Reply:
OK. I suppose in the theoretical case where we make some sort of breaking change which removes `country` from the `account` object, this isn't where we'd break anyway.